### PR TITLE
Add tests for gift cards

### PR DESCRIPTION
**Pull request by karola312**
I want to merge this change because I added tests for activate/disable multiple gift cards, purchase gift card as product and I updated existing tests for gift cards

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://qa.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-5475 Update tests for gift cards](https://app.clickup.com/t/2549495/SALEOR-5475)

File: cypress/integration/catalog/giftCards/purchaseGiftCard.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+import { createWaitingForCaptureOrder } from "../../../support/api/utils/ordersUtils";
+import * as productsUtils from "../../../support/api/utils/products/productsUtils";
+import {
+  createShipping,
+  deleteShippingStartsWith
+} from "../../../support/api/utils/shippingUtils";
+import filterTests from "../../../support/filterTests";
+
+filterTests({ definedTags: ["all"], version: "3.1.0" }, () => {
+  describe("Gift cards in checkout", () => {
```
Comment (eCzerniak) [inline cypress/integration/catalog/giftCards/purchaseGiftCard.js:15]:
```suggestion
  describe("As a customer I should be able to purchase gift card as a product", () => {
```

File: cypress/integration/catalog/giftCards/updatingGiftCards.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import {
+  createGiftCard,
+  getGiftCardWithId
+} from "../../../support/api/requests/GiftCard";
+import { deleteGiftCardsWithTagStartsWith } from "../../../support/api/utils/catalog/giftCardUtils";
+import filterTests from "../../../support/filterTests";
+import { formatDate } from "../../../support/formatData/formatDate";
+
+filterTests({ definedTags: ["all"], version: "3.1.0" }, () => {
+  describe("Tests for gift cards", () => {
```
Comment (eCzerniak) [inline cypress/integration/catalog/giftCards/updatingGiftCards.js:18]:
```suggestion
  describe("As an admin I want to update gift card", () => {
```

File: cypress/elements/giftCard/giftCardList.js [hunk 3]
```diff
@@ -1,3 +1,9 @@
 export const GIFT_CARD_LIST = {
-  issueCardButton: '[data-test-id="issueCardButton"]'
```
Comment (eCzerniak) [inline cypress/elements/giftCard/giftCardList.js:2]: Can we move this file to `cypress/elements/catalog/giftCards/giftCardList.js`  or smg like this?

**Review by eCzerniak — CHANGES_REQUESTED** [review]
Also, tests for `createGiftCards.js` are not working because correct selector is
`[data-test-id="issue-card-button"]` 
but we are using an old one:
`[data-test-id="issueCardButton"]`

**Review by bmigirl — APPROVED** [review]

File: cypress/elements/giftCard/giftCardList.js (hunk 3 above)
Comment (eCzerniak) [reply cypress/elements/giftCard/giftCardList.js:2]: @karola312 giftCards are in the catalog section I mean that maybe we should create folder in the catalog and put there files with giftCardList.js. 
` cypress/elements/giftCards/giftCardList.js`  ->  `cypress/elements/catalog/giftCards/giftCardList.js `

**Review by eCzerniak — CHANGES_REQUESTED** [review]

File: cypress/integration/catalog/giftCards/activateGiftCards.js [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+            userEmail: email,
+            initialBalance: giftCardData.amount
+          });
+        })
+        .then(dataAsExpected => {
+          expect(dataAsExpected).to.be.true;
+        });
+    });
+
+    it("should not be able to disable gift card and use it in checkout. TC: SALEOR_1006", () => {
```
Comment (eCzerniak) [inline cypress/integration/catalog/giftCards/activateGiftCards.js:153]:
```suggestion
    it("should not be able to disable gift card and use it in checkout. TC: SALEOR_1007", () => {
```

**Review by eCzerniak — APPROVED** [review]

**Pull request merged**